charts/core/templates/artifact-backend/configmap.yaml

```diff
@@ -78,4 +78,13 @@ data:
     mgmtbackend:
       host: {{ template "core.mgmtBackend" . }}
       publicport: {{ template "core.mgmtBackend.publicPort" . }}
-      privateport: {{ template "core.mgmtBackend.privatePort" . }}
\ No newline at end of file
+      privateport: {{ template "core.mgmtBackend.privatePort" . }}
+    pipelinebackend:
+      host: {{ template "core.pipelineBackend" . }}
+      publicport: {{ template "core.pipelineBackend.publicPort" . }}
+      privateport: {{ template "core.pipelineBackend.privatePort" . }}
+    milvus:
+      host: {{ template "core.milvus" . }}
```

Comment:
hi @Yougigun, I think the hostname for `milvus` I added as `core.milvus` is wrong, artifact-backend with helm deployment keep crashing and complaining not finding milvus.
there isn't a pod named `core-milvus`, I am not quite sure if we should point to the milvus proxy pod instead? can you help test it?